docs/development.rst

```diff
@@ -306,23 +317,16 @@ To connect to the database specified in ``DATABASE_URL``, use::
 
 Asset generation
 ----------------
-Kuma will automatically run in debug mode, with the ``DEBUG`` setting
-turned to ``True``. That will make it serve images and have the pages
-formatted with CSS automatically.
-
-Setting ``DEBUG=false`` file will put the installation in production mode and
-ask for minified assets.  This only works in the Vagrant environment, which
-uses Apache to serve the static files.  In Docker, static files will not be
-served and the site will be unstyled.
-
-Production assets
-*****************
-Assets are compressed on production. To emulate production and test compressed
-assets locally (*Vagrant only*):
-
-#. Set the environment variables ``DEBUG=false``.
-#. Run ``make compilejsi18n collectstatic`` in the VM or container.
-#. Restart the web process by restarting ``foreman``.
+Kuma will automatically run in debug mode, with the ``DEBUG`` setting turned to
+``True``. Setting ``DEBUG=False`` will put you in production mode and
+generate/use minified (compressed) and versioned (hashed) assets. To
+emulate production, and test compressed and hashed assets locally:
+
+#. Set the environment variable ``DEBUG=false``.
```

Comment:
After a little local testing, it appears that the `DEBUG` setting in `docker-compose.yml` is respected, while the same setting in `.env` is not (at least in terms of working with assets).